lib/flutter_stories.dart

```diff
@@ -1,5 +1,6 @@
 library flutter_stories;
 
+import 'package:flutter/material.dart';
```

Comment:
I think it's a good practice to not depend on either `material` or `cupertino` package when creating widget libs